build/commands/lib/config.js

```diff
@@ -29,36 +29,17 @@ const run = (cmd, args = []) => {
   return prog
 }
 
-// this is a huge hack because the npm config doesn't get passed through from brave-browser .npmrc/package.json
```

Comment:
why is this removed? Doesn't seem like any of this would work without this function?

Reply:
oops